addons/mail/static/src/components/attachment_viewer/attachment_viewer.xml

```diff
@@ -23,11 +23,12 @@
                 <div class="o_AttachmentViewer_headerItem o_AttachmentViewer_name">
                     <t t-esc="attachmentViewer.attachment.displayName"/>
                 </div>
-                <div class="o_AttachmentViewer_buttonDownload o_AttachmentViewer_headerItem o_AttachmentViewer_headerItemButton" t-on-click="_onClickDownload" role="button" title="Download">
+                <div class="o-autogrow"/>
+                <div class="btn btn-sm btn-primary o_AttachmentViewer_buttonDownload o_AttachmentViewer_headerItem o_AttachmentViewer_headerItemButton" t-on-click="_onClickDownload" role="button" title="Download">
                     <i class="fa fa-download fa-fw" role="img"/>
+                    <span class="d-none d-md-inline">Download</span>
```

Comment:
No `d-none d-md-inline` in template. If it shouldn't be displayed in some device config, have proper `t-if`.